Userland/DevTools/Profiler/SourceModel.cpp

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2021, Stephan Unverwerth <s.unverwerth@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "SourceModel.h"
+#include "Profile.h"
+#include <LibDebug/DebugInfo.h>
+#include <LibGUI/Painter.h>
+#include <LibSymbolication/Symbolication.h>
+#include <stdio.h>
+
+namespace Profiler {
+
+static constexpr StringView const c_source_root_path = "/usr/src/serenity/"sv;
```

Comment:
This does not have to be marked as const, if it is constexpr

Reply:
Done!